gatsby-node.js

```diff
@@ -21,23 +21,3 @@ exports.createPages = async ({ graphql, actions }) => {
   await createBlogPost({ graphql, createPage });
   await createTagPage({ graphql, createPage });
 };
-
-exports.onCreateWebpackConfig = ({ actions }) => {
```

Comment:
This was there since the beginning, when the posts were in `.md`. Not it isn't necessary anymore